# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tcsh/undef_tiocgltc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


This patch is needed on Alpha AXP ...

--- ./ed.h.orig	Sat Nov 25 09:42:59 2000
+++ ./ed.h	Sat Nov 25 09:43:03 2000
@@ -37,6 +37,10 @@
 #ifndef _h_ed
 #define _h_ed
 
+#ifdef TIOCGLTC
+# undef TIOCGLTC
+#endif
+
 #ifndef EXTERN
 # define EXTERN extern
 #endif
